google/resource_containeranalysis_note.go

```diff
@@ -0,0 +1,291 @@
+// ----------------------------------------------------------------------------
+//
+//     ***     AUTO GENERATED CODE    ***    AUTO GENERATED CODE     ***
+//
+// ----------------------------------------------------------------------------
+//
+//     This file is automatically generated by Magic Modules and manual
+//     changes will be clobbered when the file is regenerated.
+//
+//     Please read more about how to change this file in
+//     .github/CONTRIBUTING.md.
+//
+// ----------------------------------------------------------------------------
+
+package google
+
+import (
+	"fmt"
+	"log"
+	"reflect"
+	"strings"
+
+	"github.com/hashicorp/terraform/helper/schema"
+)
+
+func resourceContainerAnalysisNote() *schema.Resource {
+	return &schema.Resource{
+		Create: resourceContainerAnalysisNoteCreate,
+		Read:   resourceContainerAnalysisNoteRead,
+		Update: resourceContainerAnalysisNoteUpdate,
+		Delete: resourceContainerAnalysisNoteDelete,
+
+		Importer: &schema.ResourceImporter{
+			State: resourceContainerAnalysisNoteImport,
+		},
+
+		Schema: map[string]*schema.Schema{
+			"attestation_authority": {
```

Comment:
This whole thing is just a single string, right?  It's a little messy for users ... is it a lot of work to collapse nested objects?

Reply:
Talked about this over chat, but for others- yes, it would be a lot of work. Right now we can rename things easily but we don't have an easy way to change the type of a field- the best we'd be able to do is a lot of custom code, which we don't want to do.